src/containers/WindowTopMenuButton.js

```diff
@@ -13,6 +13,9 @@ const styles = theme => ({
   ctrlBtn: {
     margin: theme.spacing.unit,
   },
+  ctrlBtnSelected: {
+    backgroundColor: theme.palette.action.selected,
```

Comment:
CodeClimate is complaining about this being duplicated.  I _could_ extract this into an abstract mirador menu icon component, but not sure if it's really worth it at this point.